Polygon_mesh_processing/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp

```diff
@@ -5,30 +5,55 @@
 #include <CGAL/Polygon_mesh_processing/orient_polygon_soup.h>
 #include <CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h>
 
+#include <algorithm>
+#include <array>
 #include <iostream>
 #include <vector>
 
 typedef CGAL::Exact_predicates_inexact_constructions_kernel     K;
+typedef K::FT                                                   FT;
 typedef K::Point_3                                              Point_3;
 
-typedef std::vector<std::size_t>                                CGAL_Polygon;
 typedef CGAL::Surface_mesh<Point_3>                             Mesh;
 
+typedef std::array<FT, 3>                                       Custom_point;
+typedef std::vector<std::size_t>                                CGAL_Polygon;
+
 namespace PMP = CGAL::Polygon_mesh_processing;
 
+struct Array_traits
```

Comment:
Since a geom traits is needed, I think I would also add in this example the point map. This gives a nice example for a user defined point. You can then undo the changes in the other file.

Reply:
But that would go against explicitly testing both functions with `std::array<FT, 3>` as point type, wouldn't it ?

Reply:
As soon as the geom_traits is needed, the old behavior is broken. But that's not an issue, the old code was somehow incorrect since it was using free functions/operators.

Reply:
The traits is there for `repair_polygon_soup`, not `polygon_soup_to_polygon_mesh`.